airflow/www/utils.py

```diff
@@ -186,9 +186,9 @@ def is_current(current, page):
                                                       showPaused=showPaused)),
                                    disabled=is_disabled))
 
-    output.append('</ul>')
+    output.append(Markup('</ul>'))
 
-    return wtforms.widgets.core.HTMLString('\n'.join(output))
+    return functools.reduce(lambda x, y: x + "\n" + y, output, '')
```

Comment:
nit: why not `return '\n'.join(output)`? (if you want a prefixing `\n`, then `return '\n'.join([''] + output)` or `return '\n' + '\n'.join(output)`) I don't see difference. In such way we don't need to import `functools` here.

Reply:
There might be a better way of doing this - but join returns a string, the functools method returns a functools.Markup object so that we know when it is interpolated in a template that this is safe html.

I could do `return Markup('\n'.join(output))` I guess

http://flask.pocoo.org/docs/1.0/api/#flask.Markup

Reply:
Done that, and added test to ensure the right kind of object is returned.